features/font-feature-settings.yml.dist

```diff
@@ -0,0 +1,18 @@
+# Generated from: font-feature-settings.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: high
+  baseline_low_date: 2017-04-05
+  baseline_high_date: 2019-10-05
+  support:
+    chrome: "48"
+    chrome_android: "48"
+    edge: "15"
```

Comment:
This is off by a couple of versions with caniuse, but that's no trouble—this is before 2020. I made of a note of it in https://github.com/web-platform-dx/web-features/issues/1499.